<!-- Launch file for Ground Filter -->
<launch>
	<arg name="subscribe_topic" default="/points_clipped" />
	<arg name="remove_ground" default="true" />
	<arg name="points_distance" default="0.2" />
	<arg name="angle_threshold" default="0.35" />
	
	<!-- rosrun lidar_tracker ground_filter -->
	<node pkg="points_preprocessor" type="ground_filter" name="ground_filter">
		<param name="subscribe_topic" value="$(arg subscribe_topic)" />
		<param name="remove_ground" value="$(arg remove_ground)" />
		<param name="points_distance" value="$(arg points_distance)" />
		<param name="angle_threshold" value="$(arg angle_threshold)" />
	</node>

</launch>
